src/main/java/org/opentripplanner/routing/core/FareType.java

```diff
@@ -10,10 +10,7 @@
 @Deprecated
 public enum FareType implements Serializable {
```

Comment:
Is this type kept for API backwards compatibility, or should it be removed?

Reply:
Yes, only for backwards-compat. It's still used in the Atlanta and ORCA calculators. Maybe it could become something that only those two use.